### PR TITLE
[PLATFORM-421] Multiple Module Selection

### PR DESCRIPTION
**Pull request by timoxley**
* Uses Selection logic from dashboard.
* Allow multiple module selection if user holds down control/shift when clicking on modules.
* Allows multiple module deletion.
* Does not yet allow multiple module drag/drop ([TODO](https://streamr.atlassian.net/browse/PLATFORM-481))

### CODE REVIEW

File: app/src/editor/shared/components/Selection.jsx [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+    }
+
+    componentWillUnmount() {
+        window.removeEventListener('keydown', this.updateKeys)
+        window.removeEventListener('keyup', this.updateKeys)
+    }
+
+    updateKeys = (event) => {
+        this.shiftKey = event.shiftKey
+        this.ctrlKey = event.ctrlKey
```
Comment (mondoreale) [inline app/src/editor/shared/components/Selection.jsx]: Do we have to keep track on both? How about:
```js
this.keep = event.shiftKey || event.ctrlKey
```

Comment (timoxley) [reply app/src/editor/shared/components/Selection.jsx]: Intention is to keep the business logic about what should happen in the `onEvent` handler. This handler just collects state but makes no decisions.

Comment (mondoreale) [reply app/src/editor/shared/components/Selection.jsx]: Cool.

**Comment by mondoreale** [comment]
@timoxley want me to take this one from you by any chance? 💃

**Comment by timoxley** [comment]
Closing for now, will reopen when revisited.

**Pull request closed without merging**